pkg/controller/replication_controller.go

```diff
@@ -72,6 +74,20 @@ func (r RealPodControl) createReplica(namespace string, controller api.Replicati
 		desiredAnnotations[k] = v
 	}
 
+	createdByRef, err := api.GetReference(&controller)
+	if err != nil {
+		util.HandleError(fmt.Errorf("unable to get controller reference: %v", err))
+		return
+	}
+
+	createdByRefJson, err := json.Marshal(createdByRef)
```

Comment:
I suppose using the latest stable API version would be better. Really, the RC controller shouldn't be using the internal rep. at all.